stop-demo.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh -e
+
+docker-compose down
+docker-compose --profile hydra-node down
```

Comment:
Why we need profiles here?

Did not know about them BTW.

Reply:
We use the profile to start the hydra-nodes after the cardano-node (even though they are all in the same dockerfile)

Reply:
Does not `depends_on` does this?

While I am a little bit out of sync with latest docker changes, so I may miss something.